diff -wpruN '--exclude=*.orig' a~/configure a/configure
--- a~/configure	1970-01-01 00:00:00
+++ a/configure	1970-01-01 00:00:00
@@ -27070,7 +27070,7 @@ if  test x"$have_threads" = xposix; then
       # Sun Studio expands -mt to -D_REENTRANT and -lthread
       # gcc expands -pthreads to -D_REENTRANT -D_PTHREADS -lpthread
       G_THREAD_CFLAGS="-D_REENTRANT -D_PTHREADS"
-      G_THREAD_LIBS="-lpthread -lthread"
+      G_THREAD_LIBS=""
       ;;
     *)
       for flag in pthread pthreads mt; do
diff -wpruN '--exclude=*.orig' a~/configure.ac a/configure.ac
--- a~/configure.ac	1970-01-01 00:00:00
+++ a/configure.ac	1970-01-01 00:00:00
@@ -2022,7 +2022,7 @@ AS_IF([ test x"$have_threads" = xposix],
       # Sun Studio expands -mt to -D_REENTRANT and -lthread
       # gcc expands -pthreads to -D_REENTRANT -D_PTHREADS -lpthread
       G_THREAD_CFLAGS="-D_REENTRANT -D_PTHREADS"
-      G_THREAD_LIBS="-lpthread -lthread"
+      G_THREAD_LIBS=""
       ;;
     *)
       for flag in pthread pthreads mt; do
